batch_convert.qml

```diff
@@ -46,19 +46,10 @@ MuseScore {
       }
     else
       window.visible = true // needed for unknown reasons
-
-  //if (mscoreSettings.globalStyle == 0 /* MuseScoreStyleType.DARK_FUSION */)
-    //window.color = "black"
-  //else // MuseScoreStyleType.LIGHT_FUSION
-    //window.color = "lightgrey"
     }
 
-//Window {
     id: window
-  //visible: true
-  //title: qsTr("Choose Formats") // How?
-  //modality: Qt.ApplicationModal // behave like a dialog
-
```

Comment:
These commented out things should rather be kept, I want the mac and the master branch be as similar as possible for easier handling og merge conflicte between the two